test/build-turbopack-tests-manifest.js

```diff
@@ -1,5 +1,11 @@
 const fetch = require('node-fetch')
 const fs = require('fs')
+const prettier = require('prettier')
+
+async function format(text) {
```

Comment:
Since committing the file will run prettier you'd get a changed file in git but then during commit it would show as an empty commit which can be confusing, this makes sure we run prettier immediately when writing the file.